### PR TITLE
Fix Debug build configuration issues

### PR DESCRIPTION
**Pull request by maximsmol**
This fixes many flaky Asserts and build configuration issues for the Debug build.

### CODE REVIEW

File: game/client/client_base.vpc [hunk 1]
```diff
@@ -71,6 +46,47 @@ $Configuration
...
+		$OutputDirectory			".\Debug_$GAMENAME" [$WINDOWS]
+		$IntermediateDirectory		".\Debug_$GAMENAME" [$WINDOWS]
+
+		$OutputDirectory			".\Debug_$GAMENAME_360"		[$X360]
+		$IntermediateDirectory		".\Debug_$GAMENAME_360"		[$X360]
+	}
+
+	$Debugging
+	{
+		$CommandArguments "-steam -game tf -insecure -novid -nojoy -nosteamcontroller -nohltv -particles 1 -noborder -particle_fallback 2 -dev -allowdebug -condebug -sw -w 800 -h 600" [$WINDOWS]
```
Comment (maximsmol) [inline game/client/client_base.vpc:70]: Not sure what you'd want to put here, this is a mix between what you use and what I use.

File: game/client/client_base.vpc [hunk 2]
```diff
@@ -71,6 +46,47 @@ $Configuration
...
+		$OutputDirectory			".\Release_$GAMENAME" [$WINDOWS]
+		$IntermediateDirectory		".\Release_$GAMENAME" [$WINDOWS]
+
+		$OutputDirectory			".\Release_$GAMENAME_360"	[$X360]
+		$IntermediateDirectory		".\Release_$GAMENAME_360"	[$X360]
+	}
+
+	$Debugging
+	{
+		$CommandArguments "-steam -game tf -insecure -novid -nojoy -nosteamcontroller -nohltv -particles 1 -noborder -particle_fallback 2 -fullscreen -w 1920 -h 1080" [$WINDOWS]
```
Comment (maximsmol) [inline game/client/client_base.vpc:88]: Not sure what you'd want to put here, this is a mix between what you use and what I use.

Comment (mastercoms) [reply game/client/client_base.vpc:88]: Remove -fullscreen -w 1920 -h 1080

File: game/client/client_base.vpc (hunk 1 above)
Comment (mastercoms) [reply game/client/client_base.vpc:70]: Remove -fullscreen -w 1920 -h 1080

**Pull request merged**